Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.65 as builder
+FROM rust:latest as builder
```

Comment:
Does slim make much difference?

Reply:
As far as I understand, `slim` does not include some packages useful for building. I'll check if it makes too much of a difference.

Reply:
`slim` required only `pkg-config` to be installed additionally. I'll switch to it.